README.md

```diff
@@ -1 +1,47 @@
-# ramen
+# Ramen
+
+Ramen is a **disaster-recovery orchestrator** for stateful applications across
+a set of peer kubernetes clusters, that are deployed and managed using
```

Comment:
If you intended the OCM part of the sentence an essential part of the definition of Ramen, then I suggest removing the two commas.  Otherwise, I suggest replacing "that" with "which. https://www.grammarly.com/blog/which-vs-that/#:~:text=In%20a%20defining%20clause%2C%20use,and%20you%20can%20use%20which.

Also,  curious to know why you chose to use OCM instead of RHACM.

Reply:
Ack!

I used OCM instead of RHACM, as OCM is the project and the latter is a RH product.